# Pump Status With Header
std_msgs/Header header    # Standard ROS header
uint8 ON = 1
uint8 OFF = 0
uint8 status